bundles/statistics/statsgrid2016/components/Legend.js

```diff
@@ -1,21 +1,25 @@
-Oskari.clazz.define('Oskari.statistics.statsgrid.Legend', function(sandbox, locale) {
+Oskari.clazz.define('Oskari.statistics.statsgrid.Legend', function(sandbox) {
     this.sb = sandbox;
-    this.locale = locale;
+    this.locale = Oskari.getMsg.bind(null, 'StatsGrid');
```

Comment:
Passing the locale better than assuming that locale has been registered to a global from testing point of view at least.